code/game/objects/items/dna_injector.dm

```diff
@@ -175,14 +175,24 @@
 	remove_mutations = list(NERVOUS)
 
 /obj/item/dnainjector/antifire
-	name = "\improper DNA injector (Anti-Fire)"
+	name = "\improper DNA injector (Anti-Temp Adaptation)"
 	desc = "Cures fire."
-	remove_mutations = list(SPACEMUT)
+	remove_mutations = list(TEMPADAPT)
 
 /obj/item/dnainjector/firemut
-	name = "\improper DNA injector (Fire)"
+	name = "\improper DNA injector (Temp Adaptation)"
 	desc = "Gives you fire."
-	add_mutations = list(SPACEMUT)
+	add_mutations = list(TEMPADAPT)
+
+/obj/item/dnainjector/antipressure
+	name = "\improper DNA injector (Anti-Pressure Adaptation)"
+	desc = "Cures fire."
```

Comment:
???

Reply:
its copied from the space adapt one which was called firemut at one point in the future

Reply:
one point in the past sorry